.github/workflows/automated_release.yml

```diff
@@ -28,31 +28,33 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - uses: nelonoel/branch-name@v1.0.1
+
       - name: Build and tag the image for testing and release
         run: >-
           docker build .
           --file Dockerfile
-          --tag docker.pkg.github.com/${IMAGE_NAME}:${GITHUB_REF##*/}
+          --tag docker.pkg.github.com/${IMAGE_NAME}:${BRANCH_NAME}
 
       - name: Create mysql test warehouse database and table
         run: >-
           docker run
           --network host
           --entrypoint ''
-          docker.pkg.github.com/${IMAGE_NAME}:${GITHUB_REF##*/}
+          docker.pkg.github.com/${IMAGE_NAME}:${BRANCH_NAME}
           python setup_test_db.py
 
       - name: Run tests against the image
         run: >-
           docker run
           --network host
           --entrypoint ''
-          docker.pkg.github.com/${IMAGE_NAME}:${GITHUB_REF##*/}
+          docker.pkg.github.com/${IMAGE_NAME}:${BRANCH_NAME}
           python -m pytest --no-cov -vx
 
       - name: Set release tag
         # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
-        run: echo "RELEASE_VERSION=$([ ${GITHUB_REF##*/} = "develop" ] && printf '%s\n' $(cat .release-version)-develop || printf '%s\n' $(cat .release-version))" >> $GITHUB_ENV
+        run: echo "RELEASE_VERSION=$(printf '%s%s\n' $(cat .release-version) $([ ${BRANCH_NAME} = "develop" ] && printf '.%s-develop' ${GITHUB_RUN_ID} || echo ""))" >> $GITHUB_ENV
```

Comment:
I presume GITHUB_RUN_ID is a variable provided in the action?? Maybe a silly question but just wanted to check.

Reply:
Yes it's provided automatically by GitHub when the workflow is run